code/datums/antagonists/datum_cult.dm

```diff
@@ -1,3 +1,5 @@
+#define SUMMON_POSSIBILITIES 3
```

Comment:
Anything is possible but my experience is nobody ever touches these kind of config settings